ci/matrix.yml

```diff
@@ -0,0 +1,29 @@
+
+cuda_oldest: &cuda_oldest '11.8'
```

Comment:
Unfortunately there isn't a prebuild CUDA11.8 rapidsai image that follows the required naming scheme.
There exists `rapidsai/devcontainers:23.08-cpp-cuda11.8-ubuntu22.04` which is missing the `-gccXX-` part in the name.

Reply:
Is there a way for us to add CI checks with nvcc 11.8?